src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -688,13 +688,13 @@ func (it *BookIterator) PageInfo() *iterator.PageInfo {
 
 // Next returns the next result. Its second return value is iterator.Done if there are no more
 // results. Once Next returns Done, all subsequent calls will return Done.
-func (it *BookIterator) Next() (*librarypb.Book, error) {
-    if err := it.nextFunc(); err != nil {
-        return nil, err
+func (it *BookIterator) Next() (item *librarypb.Book, err error) {
```

Comment:
Can you revert to the previous implementation? They seem identical in behavior (unless I'm missing something) and the original is more idiomatic Go.